docs/src/pages/components/loader/LoaderPropControls.tsx

```diff
@@ -56,6 +74,28 @@ export const LoaderPropControls: React.FC<LoaderPropControlsProps> = ({
         value={filledColor}
         onChange={(event) => setFilledColor(event.target.value)}
       />
-    </DemoBox>
+      {isDeterminate ? (
```

Comment:
Nit: it might be better to put this below the `isDeterminate` switch so that if you switch it on/off the switch itself doesnt move.